libs/scraper-archive/src/testData.ts

```
@@ -180,3 +227,21 @@ function scoreArgs(arg: any, clean: any) {
+  const meta = test.snapshots(name)[0];
+  if (!meta) {
+    return true;
+  }
+  const snapshot = getSnapshot(meta);
+  if (snapshot?.found?.data === undefined) {
+    const t2 = getSnapshot(meta);
+    console.log("Snapshot not found for", test.key, name, t2);
+    return false;
+  }
+  return (
+    elm?.data.text == snapshot.found.data.text &&
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)